code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -314,7 +314,7 @@
 	New()
 		src.modules += new /obj/item/device/flashlight(src)
 		src.modules += new /obj/item/device/flash/cyborg(src)
-		src.modules += new /obj/item/weapon/pen/robopen(src)
+		src.modules += new /obj/item/weapon/pen/multi/robopen(src)

```

Comment:
I like that getting emagged was the only way for it to be able to stamp things at all, and then it was given the least useful stamp.